docs/user-guide/creating-application/deployment-template/rollout-deployment.md

```diff
@@ -71,6 +71,9 @@ LivenessProbe:
   successThreshold: 1
   timeoutSeconds: 5
   failureThreshold: 3
+  command:
```

Comment:
Why is this added? Please explain in the description of the PR.

Reply:
No need to change

Reply:
Use
```
  command:
    - python
    - /etc/app/healthcheck.py
   ```